### PR TITLE
[Fix](Full compaction) Fix local variable using in test case

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

In this case, some variables use global variables, causing test errors during concurrency.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

File: regression-test/suites/compaction/test_full_compaction_by_table_id.groovy [hunk 1]
```diff
@@ -120,12 +120,12 @@ suite("test_full_compaction_by_table_id") {
...
-            table_id = tablet_info[0][5]
-            backend_id = tablets[i][2]
-            times = 1
+            def table_id = tablet_info[0][5]
+            def backend_id = tablets[i][2]
+            def times = 1
 
             do{
-                (code, out, err) = be_run_full_compaction_by_table_id(backendId_to_backendIP.get(backend_id), backendId_to_backendHttpPort.get(backend_id), table_id)
+                def (code, out, err) = be_run_full_compaction_by_table_id(backendId_to_backendIP.get(backend_id), backendId_to_backendHttpPort.get(backend_id), table_id)
```
Comment (shuke987) [inline regression-test/suites/compaction/test_full_compaction_by_table_id.groovy:128]: pls fix this at line 108 too

**Comment by Yukang-Lian** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**